jsonnet/components/prometheus-adapter-audit.libsonnet

```diff
@@ -0,0 +1,117 @@
+// NOTE: audit rules are copied from openshift/library-go/pkg/operator/apiserver/audit/manifests/*.yaml
+// to maintain the same UX as OpenShift apiserver
+
+// createProfile prepends base audit rules to the rules passed
+local createProfile(rules) =
+  // see openshift/library-go/pkg/operator/apiserver/audit/manifests/base-policy.yaml
+  local baseAuditPolicy = |||
+    apiVersion: audit.k8s.io/v1
```

Comment:
Can't this be defined as jsonnet and later convert to string(for configmap) using std.toString()? It bit odd to see k8s objects being defined as string inside jsonnet.

Reply:
`data` part of the configmap which is a string is represented like this everywhere in jsonnet .e.g. PTAL at prometheus-config.yaml. This configmap's data is storing an object/configuration that isn't an actual k8s resource but is an object that encoded/decoded using k8s schema and encoders.